### ISSUE TITLE
Angular 7 

### ISSUE THREAD

**Issue by tomadj** [scaljeri/angular-xxl#2]
Hello ! 

Can you provide support for angular 7 ? 

Thank's 

Thomas

**Comment by scaljeri** [comment]
Do you experience any issues with 7 ?

**Comment by tomadj** [comment]
Rather with rxjs 6.  
But it worksk after install rxjs-compat ! 

Thank's

**Comment by scaljeri** [comment]
I've release v1.0.2 with all dependencies updated

**Comment by thorn0** [comment]
Doesn't work for me with Angular 7.1.3:
```console

ERROR in node_modules/angular-xxl/bundle/route.decorators.d.ts(1,10): error TS2305: Module '"D:/dev/router-experiments/node_modules/rxjs/src/interfaces"' has no exported member 'OperatorFunction'.
node_modules/rxjs/src/interfaces.ts(1,15): error TS2307: Cannot find module 'rxjs-compat/interfaces'.

```

**Comment by scaljeri** [comment]
Did you install `rxjs-compat`?

**Comment by scaljeri** [comment]
Good point. Its fixed in v1.07

**Comment by scaljeri** [comment]
Because the issue is fixed now, I'm going to close this it
